cli/cmd/inject_test.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+)
+
+func TestInjectYAML(t *testing.T) {
+	t.Run("Run successful conduit inject on valid k8s yaml", func(t *testing.T) {
+		file, err := os.Open("testdata/inject_emojivoto_deployment.input.yml")
+		if err != nil {
+			t.Errorf("error opening test file: %v\n", err)
+		}
+
+		goldenFileBytes, err := ioutil.ReadFile("testdata/inject_emojivoto_deployment.golden.yml")
+		if err != nil {
+			t.Fatalf("Unexpected error: %v", err)
+		}
+		expectedContent := string(goldenFileBytes)
```

Comment:
NIT 1: It seems better to me to read the golden file later, after we've done the "Unexpected error" test and right above the check that the expected output is the same as the actual output.

NIT 2: This variable is named `expectedContent` and but the other variable is named `actualOutput`. I suggest using parallel naming: `actualOutput` and `expectedOutput`.